pyglet/media/drivers/xaudio2/lib_xaudio2.py

```diff
@@ -243,43 +243,43 @@ class XAUDIO2_VOICE_DETAILS(Structure):
 class IXAudio2Voice(com.pInterface):
     _methods_ = [
         ('GetVoiceDetails',
-         com.STDMETHOD(POINTER(XAUDIO2_VOICE_DETAILS))),
+         com.VOIDMETHOD(POINTER(XAUDIO2_VOICE_DETAILS))),
         ('SetOutputVoices',
-         com.STDMETHOD()),
+         com.STDMETHOD(POINTER(XAUDIO2_VOICE_SENDS))),
         ('SetEffectChain',
          com.STDMETHOD(POINTER(XAUDIO2_EFFECT_CHAIN))),
         ('EnableEffect',
-         com.STDMETHOD()),
+         com.STDMETHOD(UINT32, UINT32)),
         ('DisableEffect',
-         com.STDMETHOD()),
+         com.STDMETHOD(UINT32, UINT32)),
         ('GetEffectState',
-         com.STDMETHOD()),
+         com.VOIDMETHOD(UINT32, POINTER(BOOL))),
         ('SetEffectParameters',
-         com.STDMETHOD()),
+         com.STDMETHOD(UINT32, c_void_p, UINT32, UINT32)),
         ('GetEffectParameters',
-         com.STDMETHOD()),
+         com.VOIDMETHOD(UINT32, POINTER(BOOL))),
         ('SetFilterParameters',
          com.STDMETHOD(POINTER(XAUDIO2_FILTER_PARAMETERS), UINT32)),
         ('GetFilterParameters',
-         com.STDMETHOD()),
+         com.VOIDMETHOD(POINTER(XAUDIO2_FILTER_PARAMETERS))),
         ('SetOutputFilterParameters',
-         com.STDMETHOD()),
+         com.STDMETHOD(c_void_p, POINTER(XAUDIO2_FILTER_PARAMETERS, UINT32))),
```

Comment:
I just noticed. This is invalid, causes the XAudio2 driver to not be used. 

`TypeError: _ctypes.POINTER() takes exactly one argument (2 given)`